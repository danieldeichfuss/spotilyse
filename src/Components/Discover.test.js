import React from "react";
import Discover from "./Discover";
import { shallow, configure } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import ReactSixteenAdapter from "enzyme-adapter-react-16";

configure({ adapter: new ReactSixteenAdapter() });
const selectedArtist = [];
const relatedArtists = {
  artists: []
};
const getRelatedArtists = jest.fn();

const wrapper = shallow(
  <Discover
    selectedArtist={selectedArtist}
    getRelatedArtists={getRelatedArtists}
    relatedArtists={relatedArtists}
  />
);
it("renders component correctly", () => {
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
