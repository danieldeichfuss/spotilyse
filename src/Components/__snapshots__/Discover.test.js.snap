// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component correctly 1`] = `
<div
  className="Discover"
>
  <h2
    className="Discover__title"
  >
    Discover your favourite Artists
  </h2>
  <Graph
    getRelatedArtists={[Function]}
    relatedArtists={Array []}
    selectedArtist={Array []}
  />
</div>
`;
